test/integration/targets/openvswitch_bridge/tests/basic.yaml

```
@@ -11,7 +11,7 @@
 
 - assert:
     that:
-      - "result.changed == true"
+      - result.changed
```

It's better to use the `changed` test. That looks through the results for a `changed` key.
```suggestion
      - result is changed
```